tests/api/content/item/invalid/duplicate.xml

```diff
@@ -1,18 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<?xml-model href="../basic.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
-<?xml-model href="../basic.rng" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<?expected-error line="13" message="Duplicate part {http://libero.pub}front@xml:lang='en'" node="/libero:item/libero:front[2]"?>
+<?xml-model href="../extra-content.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="../extra-content.rng" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<?expected-error line="14" message="Duplicate part {http://libero.pub/test}content@xml:lang='en'" node="/libero:item/test:content[2]"?>
 
-<libero:item xmlns:libero="http://libero.pub">
+<libero:item xmlns:libero="http://libero.pub" xmlns:test="http://libero.pub/test">
 
-    <libero:front xml:lang="en">
-        <libero:id>1234</libero:id>
-        <libero:title>Title</libero:title>
-    </libero:front>
+    <test:content xml:lang="en"/>
 
-    <libero:front xml:lang="en">
+    <libero:meta>
         <libero:id>1234</libero:id>
```

Comment:
may be useful to have a `service` in here?

Reply:
Probably. The ID's already duplicated in the URI so might as well do this too.